### DISCUSSION TITLE
Ability select optionLabel and optionValue from nested object

### DISCUSSION THREAD

**Discussion by vinodsptharsha** [azeezat/react-native-select#47] [Ideas]
**Description.**
Ability to select optional labels and option values from the nested object

Example, I have data that looks like below,
```javascript
const data = [
{id: "dfgdudbhkio", name: {en: "Bengaluru",  kn: "ಬೆಂಗಳೂರು"} },
{id: "kjhvhssfsdf", name: {en: "Mysore", kn: "ಮೈಸೂರು"} },
]
``` 
I can't set name.en as optionLabel from the above array. 

**The solution I'd like**
I should be able to set optionLabel and optionValue from the nested object data

**Alternatives I've considered**
Currently, I am flattening the nested object field to set optionLabel

**Additional context**
The same can applied to optionValue as well.

**Comment by azeezat** [comment]
It is important to bear in mind that arrays offer an abundance of possibilities, making it challenging to cater to every individual use case. I'm eager to hear everyone's input and perspectives on this matter.
